hummingbot/connector/exchange/zebpay/zebpay_api_order_book_data_source.py

```diff
@@ -101,24 +101,35 @@ async def get_last_traded_price(cls, trading_pair: str, base_url: str = "https:/
     @classmethod
     @cachetools.func.ttl_cache(ttl=10)
     def get_mid_price(cls, trading_pair: str, domain=None) -> Optional[Decimal]:
```

Comment:
BTW, this method may not be necessary anymore, as I noticed the HB guys are currently removing it from the other connectors... But let's keep it until we can learn what is going on, hopefully when we push the idex connector for them to review.

Reply:
Sounds good!